docs/examples/commandcollections.rst

```diff
@@ -0,0 +1,27 @@
+.. _example_commandcollections:
+
+Documentating |CommandCollection|
+=================================
+
+The client in the file ``examples/commandcollections/cli.py`` using a
```

Comment:
I don't think we need to reference the file when we're dumping the contents here. How about simply:

    Consider the following sample application, using |CommandCollection|: